src/create/aws/create-http-fallback/write.js

```diff
@@ -0,0 +1,55 @@
+let aws = require('aws-sdk')
+
+/**
+ * haha, wow
+ */
```

Comment:
im gonna print the diff of the above three lines out, frame it and put it up on my wall